pkg/models/resources/resources.go

```diff
@@ -142,9 +143,9 @@ func ListResources(namespace, resource string, conditions *params.Conditions, or
 		return nil, err
 	}
 
-	for i, d := range result {
+	for i, item := range result {
 		if i >= offset && (limit == -1 || len(items) < limit) {
-			items = append(items, d)
+			items = append(items, injector.addExtraAnnotations(item))
```

Comment:
I think it's better to move add annotations part to `persistentvolumeclaims.go`, for now, only pvc needed to be dealt with differently